packages/mdc-chips/chip/mdc-chip.scss

```diff
@@ -16,17 +16,19 @@
 
 @import "@material/ripple/common";
 @import "@material/ripple/mixins";
+@import "../mixins";
+@import "../variables";
 
 .mdc-chip {
   @include mdc-ripple-surface;
   @include mdc-ripple-radius-bounded;
   @include mdc-states;
+  @include mdc-chip-corner-radius($mdc-chip-border-radius-default);
+  @include mdc-chip-fill-color($mdc-chip-fill-color-default);
+  @include mdc-chip-ink-color($mdc-chip-ink-color-default);
```

Comment:
Remove the `mdc-states` include 3 lines above this, since now it's handled within the `ink-color` mixin.